### ISSUE TITLE
PS 8.0.4 install bug - Table '*.*_shop_group' doesn't exist

### ISSUE THREAD

**Issue by geimsdin** [PrestaShop/PrestaShop#32678]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Trying to install a fresh new PS 8.0.4 on Cyberpanel - Openlitespeed - Ubuntu 20.04 - I get the error "Table '*.*_shop_group' doesn't exist"

checking the install/data/db_structure.sql file I see that the table is not created anywhere, I don't know if that can be the issue or it is supposed to be created elsewhere, as far as I can tell seems that is the only db structure creation file in the installation process.

![image](https://github.com/PrestaShop/PrestaShop/assets/46976508/d8ea305e-f4cc-4567-b3ee-a119d4f52145)

### Expected behavior

Create the required table and install correctly PS.

### Steps to reproduce

Just start an installation 

### PrestaShop version(s) where the bug happened

8.0.4

### PHP version(s) where the bug happened

8.0.28

### Your company or customer's name goes here (if applicable).

Unusualdope

**Comment by hibatallahAouadni** [comment]
Hello @geimsdin 

There could be many things that prevent the install from occurring successfully, but this issue is not PS Core issue. 
So, these are the most common causes to crash the install:

1. Network connexion is too slow
2. The server is too restrictive, and does not allow for long running processes.  You need to increase the max execution time (if the host allows you to)

We will close this issue.

Thank you for your understanding.

**Comment by geimsdin** [comment]
Hi @hibatallahAouadni ,

very fast in closing the issue. None of your generic suggestions are appliable to this case.

I switched my php version to 7.4 and magically it worked.

On the same server I have other 2 Prestashop 1.7.x running, installed without any problem. Using PHP 8.0.28 with execution time limit set to 3000 seconds (I manage my server), on logs I don't get any timeout error but an error related to "theme_name" var:
[STDERR] PHP Warning:  Attempt to read property "theme_name" on null in /home/***.***.***/public_html/var/cache/prod/ContainerBUTMoLX/appAppKernelProdContainer.php on line 2705 

Closing an issue without even asking some more specific infos to understand if it is a case occurring only in certain configurations is not really in the spirit of an Open Source  product where people are trying to offer their contribution in debugging, in my case, the installation process.

Thanks anyway.

Simone
